crates/ide-assists/src/handlers/bool_to_enum.rs

```diff
@@ -517,6 +539,30 @@ mod tests {
 
     use crate::tests::{check_assist, check_assist_not_applicable};
 
+    #[test]
+    fn parameter_with_usage() {
+        check_assist(
+            bool_to_enum,
+            r#"
+fn function($0foo: bool) {
```

Comment:
add tests for different param location (the second param, the last param,....) would be nice

Reply:
especially one test for closure params would be good